Resources/Prototypes/Entities/Objects/Specific/Hydroponics/nymph_seed.yml

```diff
@@ -0,0 +1,35 @@
+- type: entity
+  parent: SeedBase
+  name: pack of dionea spores
```

Comment:
calling these "dionae spores" isn't really accurate considering they, besides their appearance, currently have 0 relations to diona besides what other servers implement. Consider just changing it to "nymph seeds/spores"